geomesa-accumulo/geomesa-accumulo-tools/bin/setup-namespace.sh

```diff
@@ -23,21 +24,25 @@ function help() {
   echo ""
   echo "Optional:"
   echo "  -p    Accumulo password for the provided username"
+  echo "  -t    Login using an existing Kerberos token, else will prompt for password"
   echo "  -g    Path of the GeoMesa distributed runtime JAR, with or without raster support."
   echo "  -d    Directory to create namespace in, default: /accumulo/classpath"
   echo "  -h    HDFS URI e.g. hdfs://localhost:54310"
   echo ""
   exit 0
 }
 
-while getopts ":u:p:n:g:h:d:help" opt; do
+while getopts ":u:p:tn:g:h:d:help" opt; do
```

Comment:
Should this be "t:n"?  (I'm unsure exactly what this is doing, just asking for sanity sake.)